metadata-ingestion/sink_overview.md

```diff
@@ -0,0 +1,28 @@
+# Sinks
+
+Sinks are **destinations for metadata**.
+In general, the sink will be defined in the recipe after the _source_ like below.
+
+```yaml
+source: ...
+
+sink:
+  type: <sink_type>
+  config: ...
+```
+
+When configuring ingestion for DataHub, you're likely to be sending the metadata to DataHub over either one of the following.
+
+- [REST (datahub-rest)](sink_docs/datahub.md#datahub-rest)
+- [Kafka (datahub-kafka)](sink_docs/datahub.md#datahub-kafka)
+- [File](sink_docs/file.md)
+- Since `acryl-datahub` version `>=0.8.33.2`, the default sink is assumed to be a `datahub-rest` endpoint.
```

Comment:
we no longer need the version callout, and also this should be a new paragraph and not a bullet point in the previous list

basically all the remain lines in this doc are related to the idea that when no sink is specified, we construct a default value according to the process described